src/contexts/ServerContext.js

```diff
@@ -428,10 +431,26 @@ export const ServerProvider = ({ children }) => {
     const loadServers = async () => {
         setIsLoadingZoneContext(true);
         let zone_report = await loadZoneReport();
+        console.log("UPDATED??? ", zone_report);
+
         if (zone_report !== undefined) {
             let resc_types = new Set();
-            let catalog_service_provider = [zone_report.data.zones[0]['icat_server']];
-            let fullServersArray = catalog_service_provider.concat(zone_report.data.zones[0]['servers']);
+
+            let new_version = zone_report.data.zones[0].hasOwnProperty('icat_server') ? false : true;
```

Comment:
i think we can key off of the schema_version instead...?   hmm, maybe that's insufficient, because 4.3.0 is also reporting v4...?

```
    "schema_version": "v3",
```

Reply:
Yep, both are reporting v4 (see screenshot below)